selfdrive/car/hyundai/carstate.py

```diff
@@ -29,6 +29,8 @@ def __init__(self, CP):
                           "GEAR_SHIFTER"
     if CP.carFingerprint in CANFD_CAR:
       self.shifter_values = can_define.dv[self.gear_msg_canfd]["GEAR"]
+    elif self.CP.carFingerprint in (HYBRID_CAR | EV_CAR):
+      self.shifter_values = can_define.dv["ELECT_GEAR"]["Elect_Gear_Shifter"]
```

Comment:
This just makes this gear `GearShifter.sport` from `GearShifter.unknown`, `wrongGear` alert still checks if it's drive. You need to add an extra gear to `create_common_events`

Reply:
@sshane Updated.